app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.add_alive()
+
+    def add_alive(self) -> None:
+        if self.health > 0 and self not in self.__class__.alive:
+            self.__class__.alive.append(self)
+        elif self.health <= 0 and self in self.__class__.alive:
+            self.__class__.alive.remove(self)
+
+    def __str__(self) -> str:
+        return (f"Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}")
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
```

Comment:
Code Style: In the `__repr__` method, you are using concatenation. It's better to use interpolation instead of concatenation. Please, change the concatenation to interpolation.